File: AR_Maptek_Conference/Assets/Scripts/Objects/Exposition.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

[System.Serializable]
public class Exposition
{
    public int date; // dia de la charla
    public string name_exposition;
    public string hour; // Horario de la charla. Formato ejemplo "08:00 am - 10:00 am"
    public int room;
    public string name_expositor;
    public Texture photo_expositor;
    public string info_expositor;
}

File: Webservice/api/objects/exposition.php
<?php
    include_once '../objects/entityBase.php';

    class Exposition extends EntityBase
    {
        private $table_name = "expositions";
    
        // object properties
        public $id;
        public $name_exposition;
        public $info_exposition;
        public $hour;
        public $date;
        public $room;
        public $name_expositor;
        public $photo_expositor;
        public $info_expositor;
    
        // Obtener charlas
        function getExpositions()
        {
            // query to insert record
            $query = "SELECT * FROM  ". $this->table_name ."";

            // prepare query
            $stmt = $this->conn->prepare($query);

            // execute query
            $stmt->execute();
        
            return $stmt;
        } 
    }
?>